src/Analyzers/Async/src/AsyncAnalyzer.csv

```diff
@@ -0,0 +1,3 @@
+System.Threading.Tasks.Task.Wait
+System.Threading.Tasks.Task<TResult>.Result
```

Comment:
Note that this is a pretty printed name. Most of the time type names you get from Roslyn in are the metadata format rather than looking like code a user would write. It might be more of a gotcha to try and avoid using metadata format which looks like `System.Threading.Tasks.Task`1.Result`

Reply:
Hm, I'm using ToDisplayString to get these did you have problems with it returning metadata formatted values?